lib/conditionals.md

```diff
@@ -5,8 +5,13 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is used to control the flow of a program once it is executed.  It does this by using 'if', 'else', and 'elseif' statements.  
```

Comment:
There is another place, related to control flow, where conditionals are commonly used. What is it?